src/app/routes.php

```diff
@@ -27,6 +27,7 @@
 
     "/talks-and-workshops/web-development/register/"    => "/contest/webdev_workshop/",
     "/talks-and-workshops/paper-presentation/register/" => "/contest/paper_presentation/",
+    "/contest/futsal/register"                          => "/contest/futsal/",
```

Comment:
/contest/futsal forwarded to Kurumi?
Why don't you just pick a namespace which does not conflict?